### PR TITLE
Update Active Record Validations Guide [ci skip]

### PR DESCRIPTION
**Pull request by bogdanvlviv**

### CODE REVIEW

File: guides/source/active_record_validations.md [hunk 1]
```diff
@@ -435,7 +435,7 @@ The default error message for this helper is _"is not included in the list"_.
 
 ### `length`
 
-This helper validates the length of the attributes' values. It provides a
+This helper validates the length of the attribute's value. It provides a
```
Comment (rafaelfranca) [inline guides/source/active_record_validations.md]: I think plural was correct here. You can pass more than one attribute

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

File: guides/source/active_record_validations.md (hunk 1 above)
Comment (bogdanvlviv) [reply guides/source/active_record_validations.md]: amended

**Comment by rafaelfranca** [comment]
I think all those text is correct since validations can receive more than one attribute so using plural is correct in this text. Thanks!

**Comment by bogdanvlviv** [comment]
You are right. Thanks for the review.

**Pull request closed without merging**